launch/launch/event_handlers/on_process_exit.py

```diff
@@ -88,8 +88,10 @@ def __init__(
             else:
                 self.__actions_on_exit = [on_exit]
 
-    def _handle(self, event: Event, context: LaunchContext) -> Optional[SomeActionsType]:
+    def handle(self, event: Event, context: LaunchContext) -> Optional[SomeActionsType]:
         """Handle the given event."""
+        super().handle(event, context)
```

Comment:
This is a bit weird, but I guess it's ok. Maybe I'd feel better if we captured the return and made sure it was none, or maybe even if there was just a comment explaining what is happening.

What if the base class had a `pre_handle()` method, that did everything except return entities which the derived classes could call with super?